src/parser.js

```diff
@@ -15,6 +15,7 @@ function parse(text, parsers, opts) {
   const parser = new engine({
     parser: {
       extractDoc: true,
+      version: opts.phpVersion,
```

Comment:
@cseufert @czosel I would love your help and suggestion on this.

The modification to `php-parser` allowing it to parse `readonly` is protected by PHP version checks.
The prettier plugin for php has a `phpVersion` option. But it is used only by the printer and not pass down to the PHP parser. 

Adding it is a simple change, but a breaking change. Are we ok with this or do we think of an other approach?

Reply:
We didn't need to do this till now because the default version of the parser was always the latest one. I'd say we can keep it that way - all we have to do is bump the default in the parser. I created a PR for it here: https://github.com/glayzzle/php-parser/pull/879